autoload/jqplay.vim

```diff
@@ -59,36 +59,48 @@ function! jqplay#scratch(mods, args) abort
         return s:error('jqplay: only one session per Vim instance allowed')
     endif
 
+    " If -n/--null-input is passed we won't need an input buffer for jq
+    let null_input = a:args =~# '-n\|--null-input' ? 1 : 0
```

Comment:
Short-form options can be combined in a single argument: `jq -Sn <filter>`. Something like `'-\S*n\|--null-input'` could take care of that

Reply:
I will add support for `--raw-input`. It's probably best to make `:Jqplay` a global command, and then just do a check if `-n` or/and `-R` have been passed. A command like `:JqplayScratch` will nevertheless be required in case the current buffer is json but someone wants to start a new session with a new json scratch buffer and not the current one.

Can you think of another case where `-n` is required? The issue you linked above suggests that it should be added whenever the `inputs` function is used.

Reply:
This means, when I run
```bash
cat foo.json | jq -n 'input'
```
the file `foo.json` is read by `jq`, but when I omit the `input` function, it won't be parsed:
```bash
cat foo.json | jq -n '.'
```
Ouput: `null`